kiwmi/desktop/output.c

```diff
@@ -279,6 +280,15 @@ output_destroy_notify(struct wl_listener *listener, void *UNUSED(data))
 
     wl_list_remove(&output->events.destroy.listener_list);
 
+    for (int i = 0; i < 4; i++) {
```

Comment:
Magic number 😿

Reply:
Fixed, I have a wariness about deploying `sizeof` on arrays, but I'll grant it's warranted here.